### DISCUSSION TITLE
Getting "Nested arrays may not be passed to whereIn method." error when linking to entries on Bard

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [statamic/cms#7308] [Troubleshooting]
Hi, everyone! I'm rebuilding the rich text editor my company's system uses with Bard.
This has been an incredible experience. I was able to add image sets, attachments, videos and a lot more in record time.
The only problem until now is this:

1. Select a word inside Bard. 
2. Click the `Link` button in the toolbar
3. Select the `Entry` radio button on the popup menu.
4. Click the folder icon to browse and select an entry.

This opens the modal to select the entry I want to link, but no entries are loaded to be selected and I'm getting a `Nested arrays may not be passed to whereIn method.` error toast.

```json
{
    "message": "Nested arrays may not be passed to whereIn method.",
    "exception": "InvalidArgumentException",
    "file": "/var/www/statamic/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php",
    "line": 1032,
    "trace": []
}
```

I'm not sure if this is a bug or a missconfiguration.

**Comment by edalzell** [comment]
Is this "standard" bard or have you customized it in any way?

**Reply by MtDalPizzol** [reply]
Hey, @edalzell . Sorry. This was all on me. I had a custom Fieldtype on the blueprint of the entry I was trying to link, but there where old entries that where causing the list to break.
